pantheon-files-daemon/FileManager1.vala

```diff
@@ -43,7 +43,7 @@ public class FileManager1 : Object {
         string cmd = "pantheon-files -t";
 
         foreach (string s in uris) {
-            cmd += (" " + s);
+            cmd += (" \"" + PF.FileUtils.sanitize_path (s) + "\"");
```

Comment:
Thanks for the review comments folks. Let me test both using David's code for replacing single quotes with the percent codes along with the existing methods from `FileUtils`. It may take me a couple of days to follow up.